examples/aws_efa/nccl_efa.yaml

```diff
@@ -0,0 +1,55 @@
+name: nccl-test-efa
+
+resources:
+  cloud: kubernetes
+  accelerators: A100:1
+  cpus: 64
+  image_id: docker:public.ecr.aws/hpc-cloud/nccl-tests:latest
+
+num_nodes: 2
+
+envs:
+  PATH: $PATH:/usr/local/cuda-12.2/bin:/opt/amazon/efa/bin:/usr/bin
+  LD_LIBRARY_PATH: /usr/local/cuda-12.2/lib64:/opt/amazon/openmpi/lib:/opt/nccl/build/lib:/opt/amazon/efa/lib:/opt/aws-ofi-nccl/install/lib:/usr/local/nvidia/lib:$LD_LIBRARY_PATH
+  NCCL_HOME: /opt/nccl
+  CUDA_HOME: /usr/local/cuda-12.2
+  NCCL_DEBUG: INFO
+  NCCL_BUFFSIZE: 8388608
+  NCCL_P2P_NET_CHUNKSIZE: 524288
+  NCCL_TUNER_PLUGIN: /opt/aws-ofi-nccl/install/lib/libnccl-ofi-tuner.so
```

Comment:
These do not seem to be needed to be in the `envs` section as this section is more for configurable envs that people may want to override with CLIs.